tests/topotests/bgp_basic_functionality_topo1/test_bgp_basic_functionality.py

```diff
@@ -1135,9 +1135,9 @@ def test_bgp_with_loopback_with_same_subnet_p1(request):
     protocol = "bgp"
     for addr_type in ADDR_TYPES:
         result = verify_fib_routes(tgen, addr_type, dut, input_dict_r1)
-        assert result is not True, "Testcase {} : Failed \n".format(tc_name)
+        assert result is not True, "Testcase {} : Failed \n"
```

Comment:
Yeah not including the actual test name is not going to make me happy when I get to try to find it when it fails.  why don't we just remove the newline?

Reply:
I'm not removing anything, I just move the `tc_name` argument to the correct place.

This is incorrect:
```
print("A {}".format(a)
"B{}".format(b))
```

This is correct:
```
print("A {}"
"B {}".format(a, b))
```